internal/progress/progress.go

```diff
@@ -47,6 +50,21 @@ func WithDebugOutput(w io.Writer) mpb.ContainerOption {
 // New creates a new Progress struct
 // mpb.ContainerOptions are forwarded
 func New(options ...mpb.ContainerOption) *Progress {
+	tw, _, err := terminal.GetSize(int(os.Stdout.Fd()))
+	if err != nil {
+		logging.Error("Could net get terminal size: %v", err)
+		tw = 120
+	}
+
+	// ensure that we have enough space by modifying the progress bar width
+	// we subtract 26 from the terminal width
+	// * 20 for name and counter to the left of bar
```

Comment:
Will the name and counter always be 20 characters?

Reply:
It is a good point. I misunderstood how this works.  It is dependent on the longest on the string.  So, it could change with localization. :(

Reply:
I think that I will add functionality to limit the name size based on the terminal.